web/dashboard/src/styles/base.css

```
@@ -0,0 +1,32 @@
+body {
+  font-family:
+    "Inter",
```

Stylelint (17.14.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)